app/scripts/metamask-controller.js

```diff
@@ -1070,21 +1104,14 @@ export default class MetamaskController extends EventEmitter {
       SnapController: this.snapController,
       NotificationController: this.notificationController,
       ///: END:ONLY_INCLUDE_IN
+      ...resetOnRestartConfig,
```

Comment:
@danjm This part of code is causing Confirm Screen to hang. if I added the controllers manually, it works. I'm not exactly sure why this is failing.